my-app/src/app/courses-page/courses-page.module.ts

```diff
@@ -1,28 +1,34 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-
-import { AddCourseButtonModule } from './add-course-button/add-course-button.module';
-import { CourseItemModule } from './course-item/course-item.module';
-import { CourseSearchModule } from './course-search/course-search.module';
-import { CoursesListModule } from './courses-list/courses-list.module';
-import { LoadMoreModule } from './load-more/load-more.module';
+import { FormsModule } from '@angular/forms';
 
 import { CoursesPageComponent } from './courses-page.component';
-
+import { HeaderModule } from '../shared/components/header/header.module';
+import { FakeLogoModule } from '../shared/components/fake-logo/fake-logo.module';
+import { LogInButtonModule } from '../shared/components/log-in-button/log-in-button.module';
+import { LogOutButtonModule } from '../shared/components/log-out-button/log-out-button.module';
+import { BreadcrumbsModule } from '../shared/components/breadcrumbs/breadcrumbs.module';
+import { CourseSearchComponent } from './course-search/course-search.component';
+import { CoursesListComponent } from './courses-list/courses-list.component';
+import { CourseItemComponent } from './course-item/course-item.component';
 
 
 
 @NgModule({
   declarations: [
-    CoursesPageComponent
+    CoursesPageComponent,
+    CourseSearchComponent,
```

Comment:
please create a separate module for each of those components. Then import those modules into this module